internal/resources/webform/webform-template.html

```diff
@@ -70,7 +70,12 @@ <h3>Request Timeout</h3>
         <button class="grpc-invoke" disabled>Invoke</button>
       </div>
       <div class="grpc-tabcontent" id="grpc-request-raw-tab">
+        <h3>Request payload</h3>
         <textarea id="grpc-request-raw-text"></textarea><br/>
+        <h3>gRPC curl</h3>
+        <div id="grpc-curl" class="grpc-curl-panel">
+          <span><pre id="grpc-curl-text"></pre></span>
+        </div>
```

Comment:
This PR is awesome, thank you.  My one question is: why the different constructions (textarea vs. pre)?
Seems like we should use the same setup and formatting for both.  I'm not strongly opinionated on which... if you like the new formatting, you could just make the CSS classname something like `grpc-request-panel` and apply it to both.

Reply:
Holy crap... I actually had no idea that the Raw Request box was even editable.....

Reply:
In that case, I wonder if we can move the `Invoke` button up above the grpcurl area?

I also wonder if there's a way to make it more obvious that the request box is editable..

Reply:
@dragonsinth    Like this?  

I am not a fan of this, looks messy. Buttons are usually at the bottom. 


![image](https://user-images.githubusercontent.com/131102/234396082-f0a53588-7169-411f-8f05-b6bc26d01e37.png)

Reply:
Seems fine to me.  I think it's weirder to separate the editable field from the button that it goes with.

Reply:
@dragonsinth pushed new changes


![image](https://user-images.githubusercontent.com/131102/234502572-9fe66b01-01b1-4870-a8fd-46a6d38b0147.png)